index.js

```diff
@@ -13,7 +14,20 @@ const localeMap = arr =>
 
 const getBabelrc = cwd => {
   try {
-    return readBabelrcUp.sync({ cwd }).babel
+    const babelrc = readBabelrcUp.sync({ cwd }).babel
+
+    if (!babelrc.env) {
+      return babelrc
+    }
+
+    const env = process.env.BABEL_ENV || process.env.NODE_ENV || 'development'
+    const concatArray = (obj, src) => {
+      if (Array.isArray(obj)) {
+        return obj.concat(src)
+      }
+    }
+
+    return mergeWith(babelrc, babelrc.env[env], concatArray)
```

Comment:
https://babeljs.io/docs/usage/babelrc/#env-option says:
> Options specific to a certain environment are merged into and overwrite non-env specific options.

Reply:
We should use `lodash.mergeWith` instead of `lodash.merge` to merge inside arrays.
cf. https://lodash.com/docs/4.17.4#mergeWith